scout/server/config.py

```diff
@@ -35,6 +35,22 @@
 # connection details for LoqusDB MongoDB database in the loqusdb config file
 #    uri=("mongodb://{}:{}@localhost:{}/loqusdb".format(MONGO_USERNAME, MONGO_PASSWORD, MONGO_PORT))
 
+CLOUD_IGV_TRACKS = [
```

Comment:
Left it there while testing some other issue?

Reply:
Indeed! But actually, let me just comment it out like the other in-config examples perhaps?